### DISCUSSION TITLE
请问现在还在支持多框架的架构吗？

### DISCUSSION THREAD

**Discussion by jaluik** [opentiny/tiny-vue#1511] [Q&A]
我从[这篇文章](https://mp.weixin.qq.com/s/TRIBQ--zP-nasLh1wy9q5w)过来的，大概看了一下代码。请问现在是不是没再做`react`、`solidjs`适配了呢？

**Comment by kagol** [comment]
@jaluik 支持多框架，React 和 Solid 框架都在适配的，目前两个库的流程都打通了，正在编写模板转换脚本，欢迎一起共建~
